stage1_fly/enter/main.go

```diff
@@ -47,10 +50,30 @@ func getRootDir(pid string) (string, error) {
 	return os.Readlink(rootLink)
 }
 
-func execArgs() error {
+// readEnv reads the environment from the env file written by stage1 run
+func readEnv() ([]string, error) {
```

Comment:
It looks like this is a good candidate to be moved to `rkt/common/environment.go`.